src/shared/codelens/localLambdaRunner.ts

```diff
@@ -661,38 +680,20 @@ async function waitForDebugPort({
     }
 }
 
-export interface RecordAttachDebuggerMetricContext {
+export interface ProvideDebuggerMetadataContext {
```

Comment:
This isn't a context interface...the only context is the telemetry service. Suggest to split into `ProviderDebuggerMetadataParameters` and `ProviderDebuggerMetadataContext`.

This isn't a regression, so it's optional to address in this PR.